src/sage/crypto/lattice.py

```diff
@@ -35,22 +36,24 @@ def gen_lattice(type='modular', n=4, m=8, q=11, seed=None,
     INPUT:
 
     - ``type`` -- one of the following strings
-        - ``'modular'`` (default) -- A class of lattices for which
-          asymptotic worst-case to average-case connections hold. For
-          more refer to [Aj1996]_.
-        - ``'random'`` -- Special case of modular (n=1). A dense class
-          of lattice used for testing basis reduction algorithms
-          proposed by Goldstein and Mayer [GM2002]_.
-        - ``'ideal'`` -- Special case of modular. Allows for a more
-          compact representation proposed by [LM2006]_.
-        - ``'cyclotomic'`` -- Special case of ideal. Allows for
-          efficient processing proposed by [LM2006]_.
-    - ``n`` -- Determinant size, primal:`det(L) = q^n`, dual:`det(L) = q^{m-n}`.
+
+      - ``'modular'`` (default) -- A class of lattices for which
+        asymptotic worst-case to average-case connections hold. For
+        more refer to [Aj1996]_.
+      - ``'random'`` -- Special case of modular (n=1). A dense class
+        of lattice used for testing basis reduction algorithms
+        proposed by Goldstein and Mayer [GM2002]_.
+      - ``'ideal'`` -- Special case of modular. Allows for a more
+        compact representation proposed by [LM2006]_.
+      - ``'cyclotomic'`` -- Special case of ideal. Allows for
+        efficient processing proposed by [LM2006]_.
+
+    - ``n`` -- Determinant size, primal: `det(L) = q^n`, dual: `det(L) = q^{m-n}`.
       For ideal lattices this is also the degree of the quotient polynomial.
     - ``m`` -- Lattice dimension, `L \subseteq Z^m`.
```

Comment:
you inserted empty lines lines 39 and 50 to separate items but not here. Shouldn't it be the same for all items of the list ?